_wallets/corazon.md

```diff
@@ -0,0 +1,26 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: corazon
+title: "Corazon"
+titleshort: "Corazon"
+compat: "hardware"
+level: 2
+platform:
+  - hardware:
+    name: hardware
+    os:
+      - name: hardware
+        text: "walletcorazon"
+        link: "https://gray.sg/collections/corazon-wallet"
+        source: "https://github.com/trezor/"

```

Comment:
This should link to the open source repository corresponding to the firmware, instead of the GitHub overview page for the Trezor organization.